src/Common/Core/Impl/Threading/IMainThread.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.Common.Core.Threading {
+    public interface IMainThread {
```

Comment:
Even in console application and service?

---
In reply to: [81241433](https://github.com/Microsoft/RTVS/pull/2408#discussion_r81241433) [](ancestors = 81241433,81223138,81221364)